src/utils/convertViews.js

```diff
@@ -0,0 +1,16 @@
+const convertViews = (views) => {
+  switch (views.length) {
+    case 4:
```

Comment:
why so many empty cases, instead if else can be used if only two conditions

Reply:
Its not empty cases, the code is same for that cases